crates/javy/src/alloc.rs

```diff
@@ -7,6 +7,21 @@ use std::ptr::copy_nonoverlapping;
 // non-zero to indicate success.
 const ZERO_SIZE_ALLOCATION_PTR: *mut u8 = 1 as _;
 
+/// 1. Allocate memory of new_size with alignment.
+/// 2. If original_ptr != 0
+///   a. copy min(new_size, original_size) bytes from original_ptr to new memory
+///   b. de-allocate original_ptr
+/// 3. return new memory ptr
```

Comment:
```suggestion
/// 3. Return new memory ptr.
```

Reply:
(I'm aware that this is not your change, but I thought why not improve it, given that we're here)